### ISSUE TITLE
2.5.5 can't select game version

### ISSUE THREAD

**Issue by Cyronlee** [StarGazer1258/BeatDrop#50]
2.5.5 can't select game version

**Comment by StarGazer1258** [comment]
Need more information than this. Please follow the bug report template.

**Comment by IEagleoneI** [comment]
same issue here

**Comment by JuanLargearm** [comment]
also have this issue

**Comment by FrangSierra** [comment]
Any fix for this?

**Comment by IEagleoneI** [comment]
> Any fix for this?

try vpn, it's connection to server issue

**Comment by StarGazer1258** [comment]
Yeah, this is likely due to being unable to connect to BeatMods and get the version list. It may be internet connectivity related. Not much I can do except store a local cache of known old version. This won't account for newer versions, however.

**Comment by AndrewTheLouis** [comment]
I'm getting the same error/bug too :(

**Comment by delbonis** [comment]
Also having this issue, I can access the website in browser but it doesn't seem to work in the application.  Also using WINE, but the issue is probably not WINE's fault here.

**Comment by laurens94** [comment]
Same issue here, is there a way to rule out that it's a connectivity issue? Like pinging a server or something?

**Comment by StarGazer1258** [comment]
It is also possible that the fetch API has run into an error while fetching the resource (which it often does.) If this is the case, there is little I can do about it. My suggestion would be to restart the application to reload the resource. 

If anyone wants to download the source and debug further, please feel free. Unfortunately, I have little to work on this anymore.

**Comment by laurens94** [comment]
Did a clean reinstall of windows and it's fixed, so i think it was a firewall or vpn issue or something

**Comment by DavidSprauel** [comment]
I'm getting the exact same issue, evcen trying to re install beat drop.
Any solutions found since that issue has been opened ?
